src/DataFramesMeta.jl

```diff
@@ -6,6 +6,8 @@ using MacroTools
 
 @reexport using DataFrames
 
+@reexport using Chain
```

Comment:
Probably better only reexport `@chain` and `@aside` explicitly, so that we don't end up exporting more symbols than anticipated if Chain.jl changes.

Reply:
Thanks. Only adding `@chain` since `@aside` is an unexported macro-flag.